kbase-extension/static/kbase/js/widgets/appWidgets2/input/selectInput.js

```diff
@@ -81,7 +81,7 @@ define([
                     }
                 );
                 if (validation.diagnosis === Constants.DIAGNOSIS.REQUIRED_MISSING) {
-                    validation.errorMessage = 'Please select a value from the dropdown.';
+                    validation.errorMessage = 'A value is required.';
```

Comment:
are there tests for this widget?

Reply:
Yes, but they didn't test the exact error text in the validation message. I can add that.

Reply:
Done!